backend/src/ml_space_lambda/authorizer/lambda_function.py

```diff
@@ -415,9 +415,9 @@ def _handle_dataset_request(request_method, path_params, user):
             return True
         else:
             # All admins can perform any action on any Group
-            if dataset.type == DatasetType.GROUP:
-                if Permission.ADMIN in user.permissions:
-                    return True
+            if (dataset.type == DatasetType.GROUP or dataset.type == DatasetType.GLOBAL) and Permission.ADMIN in user.permissions:
```

Comment:
I want to get this out as quickly as possible but in the future we should probably do something like this…

```suggestion
            if dataset.type in [DatasetType.GROUP, DatasetType.GLOBAL] and hasPermission(Permission.ADMIN, user.permissions):
```

We can't apply this directly because this file didn't import `hasPermission`